javagen/src/main/java/com/azure/autorest/util/ModelExampleUtil.java

```diff
@@ -106,74 +106,71 @@ public static ExampleNode parseNode(IType type, IType wireType, Object objectVal
             }
         } else if (type == ClassType.Object) {
             node = new ObjectNode(type, objectValue);
-        } else if (type instanceof ClassType && objectValue instanceof Map) {
+        } else if (type instanceof ClassType && ClientModelUtil.getClientModel(((ClassType) type).getName()) != null
+                && objectValue instanceof Map) {
             ClientModel model = ClientModelUtil.getClientModel(((ClassType) type).getName());
-            if (model != null) {
```

Comment:
Personally I still prefer call `ClientModelUtil.getClientModel` only once. This is at least `O(n)` op.

Reply:
Changed.